db_test.go

```diff
@@ -695,6 +695,32 @@ func TestWALFlushedOnDBClose(t *testing.T) {
 	testutil.Equals(t, []string{"labelvalue"}, values)
 }
 
+func TestWALSegmentSizeOption(t *testing.T) {
+	options := *DefaultOptions
+	options.WALSegmentSize = 2 * 32 * 1024 //one segment file must contains 2 pages max (32 Kb each)
```

Comment:
```suggestion
	options.WALSegmentSize = 2 * 32 * 1024
```

don't think we need a comment here, but need to add it to the help text for the flag in Prometheus. 
Something like: "...should be a delimiter of the current WAL page size which is 32kb".